switchcraft/data/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -23,9 +23,11 @@ local PASTE_ID_PATTERNS = {
 local function printUsage()
   local programName = arg[0] or fs.getName(shell.getRunningProgram())
   print("Usages:")
-  print(programName .. " put <filename>")
-  print(programName .. " get <code> <filename>")
+  print(programName .. " put <filename> [password] [visibility]")
+  print(programName .. " get <code> <filename> [password]")
```

Comment:
I'm not a fan of this CLI. When we initially wrote the pastebin program our eventual goal was to support the settings API and flag arguments for the more complex parameters of SCPaste - API tokens, visibility, etc. We still haven't done that, but I don't think we should be adding more positional arguments in the meantime. This is an irreversible change that makes future extensibility even harder.

The `visibility` option should be removed anyway (see later comment), but I think the password option should be a flag instead. This likely means adding a getopt-style argument parser to the ROM, e.g.:

https://github.com/emmachase/Riko4/blob/master/scripts/lib/getopt.lua
https://github.com/SquidDev-CC/metis/blob/dev/src/metis/argparse.lua

(double check the licenses of these programs first)